hbase-rest/pom.xml

```diff
@@ -358,6 +358,8 @@
           </systemPropertyVariables>
         </configuration>
       </plugin>
+      <!-- The protobuf-maven-plugin and com.google.code.maven-replacer-plugin config
```

Comment:
Could we move this to parent pom too? So we do not need to copy paste it everywhere...

Reply:
I will try to.

Reply:
I have centrilzed the replace plugin.
The protobuf compiler config cannot be centralized, as 2.x uses both protobuf 2.5 and thirdparty compilers.